eureka-client/src/main/java/com/netflix/discovery/shared/transport/jersey/JerseyEurekaHttpClientFactory.java

```diff
@@ -244,6 +276,14 @@ private void addFilters(ApacheHttpClient4 discoveryApacheClient) {
             String ip = myInstanceInfo == null ? null : myInstanceInfo.getIPAddr();
             AbstractEurekaIdentity identity = clientIdentity == null ? new EurekaClientIdentity(ip) : clientIdentity;
             discoveryApacheClient.addFilter(new EurekaIdentityHeaderFilter(identity));
+
+            if (additionalFilters != null) {
```

Comment:
Do we have any integration test that creates a custom filter? If not, it may be worth to add it.